easybuild/easyblocks/e/easybuildmeta.py

```diff
@@ -159,8 +159,7 @@ def sanity_check_step(self):
         for tool in eb_dirs.keys():
             self.log.debug("Trying %s.." % tool)
             try:
-                exec "from %s import setup" % tool
-                del setup
+                exec("from %s import setup" % tool)
```

Comment:
It never was really needed, it's cleanup, but it fails with Python 3, and it's tricky to get right (because you have to try hard to make `exec` actually define things in the current scope)